index.html

```diff
@@ -61,5 +70,166 @@ <h1>Hey There.<br>I'm Daniel</h1>
             </div>
         </div>
     </section>
+
+    <section class="work-section">
+        <div class="container">
+        <div class="title">
+            <h1><span>My Recent work</span></h1>
+            <img src="images/divider.png" alt=""/>
```

Comment:
Hi there, its a good job you are doing on the project! I have some suggestions to make on this image

<img width="280" alt="hy" src="https://user-images.githubusercontent.com/4247740/143719618-5b8afb05-83e3-424c-8ae6-6880163915a0.PNG">

It is slightly pushed to the right, this happens because of the style you added to it.
```
.skills span, img {
    color: white;
    margin-left: 20px; 👈🏻 
    font-size: 32px;
    font-family: 'Inter', sans-serif;
}
```
How about removing it or setting it to zero, whatever you may prefer, will align the divider in the center as seen in the figma design.

- [ ] Please make this change for the other divider in the next section.